README.md

```diff
@@ -19,6 +19,7 @@ Simply press a key (`<C-l>` in insert mode by default) and select what you want
 ### Automatic File Link Insertion
 You can use `Telescope neorg insert_file_link` to insert a link to a neorg file.
 Notice that this only works for files in the current workspace.
+Note: the prompt will be used if no selection is chosen.
```

Comment:
This isn't very clear imo
You should use something like `Note: If no file is selected a link to a file with the name of the prompt value will be inserted. This file will be created if you use the link`